### PR TITLE
Nbo/allow dot in scenario names

### PR DESCRIPTION
**Pull request by nborde-CSM**
Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [ ] Testing
- [ ] No error logs in web console (except iFrame errors)
- [ ] Create or Update documentation (README file for users/integrators)
- [ ] Identify breaking changes with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)
- [ ] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature
- [ ] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -191,7 +191,7 @@
                 "label": "Error",
                 "empty": "Scenario name cannot be empty",
                 "existing": "Scenario name already exists",
-                "forbiddenchars": "Scenario name has to start with a letter, and can only contains letters, digits, spaces, underscores and hyphens."
+                "forbiddenchars": "Scenario name has to start with a letter, and can only contains letters, digits, spaces, underscores hyphens and dots."
```
Comment (csm-thu) [inline public/locales/en/translation.json:194]: Could you add a comma between "underscores" and "hyphens" ?

File: src/utils/ValidationUtils.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 
-export const NAME_VALIDATOR = /^\w[\w\d\s-]*$/;
+export const NAME_VALIDATOR = /^\w[\w\s.-]*$/;
```
Comment (csm-thu) [inline src/utils/ValidationUtils.js:4]: Le fait de supprimer `\d` c'est volontaire ?

Comment (nborde-CSM) [reply src/utils/ValidationUtils.js:4]: Oui, \w inclut \d

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**